packages/react-scripts/template/README.md

```diff
@@ -771,6 +772,73 @@ Please refer to these two threads for reference:
+
+Start by creating a React app using `create-react-app` command line tool. Then, install the following packages:
+- `electron` is the Electron package that includes all necessary modules to create an Electron app
+- `electron-packager` is a command line tool necessary to bundle an Electron app
+- `concurrently` is a command line tool that runs two processes concurrently
+- `wait-on` is a command line tool that waits for a file, port, socket, or http resource to be available
+
+At the root of the React application folder you will need to create an Electron script for initializing the Electron app. [Electron's quick start](https://github.com/electron/electron-quick-start) `main.js` is a good starting point. Extra logic needs to be added to the window creation in order to render the React application in both development and bundled environments. On easy way to distinguish development from bundled environment is to use an environment variable. Here is an example implementation using `ELECTRON_DEV` environment variable.
+
+```javascript
+// Create the browser window.
+mainWindow = new BrowserWindow({width: 800, height: 600})
+
+// and load the index.html of the app.
+mainWindow.loadURL(
+  process.env.ELECTRON_DEV
+    ? 'http://localhost:3000' // Dev server ran by react-scripts
+    : `file://${path.join(__dirname, '/build/index.html')}` // Bundled application
+);
+```
+
+Once this file is created, the `package.json` needs to be edited:
+- Set the Electron script (here `main.js`) as `main`
+- Set application's root (`./`) as `homepage`
+- Rename `start` and `build` scripts as `react-start` and `react-build`
+- Prefix `react-start` script with `BROWSER=none` to prevent the script from opening a browser window on start
+- Add `electron-start` script that runs Electron in development mode: `ELECTRON_DEV=1 electron .`
+- Add `electron-build` script that bundles the Electron app: `electron-packager ./`
+- Add high level `start` script that runs concurrently the React development server and the Electron app once the React app is ready to be rendered: `concurrently 'yarn react-start' 'wait-on http://localhost:3000/ && yarn electron-start'`
+- Add high level `build` script that bundles both the React app and the Electron app: `yarn react-build && yarn electron-build`
+
+The `package.json` should be as follow:
+
+```JSON
+{
+  "name": "my-electron-app",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "concurrently": "^3.4.0",
+    "electron": "^1.6.1",
+    "electron-packager": "^8.5.2",
+    "react-scripts": "0.9.3",
+    "wait-on": "^2.0.2"
+  },
+  "dependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "main": "main.js",
+  "homepage": "./",
+  "scripts": {
+    "react-start": "BROWSER=none react-scripts start",
+    "electron-start": "ELECTRON_DEV=1 electron .",
```

Comment:
The `BROWSER` env variable can easily be move to a `.env` file but the `ELECTRON_DEV` flag can be problematic because for the user it means changing the env variable value when building or having multiple `.env` files and some loading logic to switch between prod and build. 

What do you think of [`electron-is-dev`](https://github.com/sindresorhus/electron-is-dev) instead of the `ELECTRON_DEV` flag?